src/search/SearchForm.tsx

```diff
@@ -1,16 +1,21 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Checkbox, Select, TextInput, Button, IconSearch } from 'hds-react';
+import { useForm } from 'react-hook-form';
 import styled from 'styled-components';
 import { breakpoint } from '../common/style';
+import { getApplicationPeriods, getParameters } from '../common/api';
+import { mapOptions, OptionType, getSelectedOption } from '../common/util';
 
-interface Props {
-  // only text search is now implemented!
-  onSearch: (text: string) => void;
-}
-interface OptionType {
-  label: string;
-}
+export type Criteria = {
```

Comment:
Maybe not needed to be exported, or if so, belongs to to the type definition file?

Reply:
fixed